backend/cn/symRel.mli

```diff
@@ -0,0 +1,89 @@
+(* Module SymPair -- TODO: BCP: What is it for?? *)
+
+(* TODO: BCP: Some filenames use_underscores, while some areInCamlCase
+   -- any compelling reason for that? *)
```

Comment:
No. We need to pick a convention and just go for it. My vote is snake_case (easier on the eyes).

Reply:
Agreed.